sysid-application/src/main/native/include/sysid/view/Analyzer.h

```diff
@@ -128,21 +152,23 @@ class Analyzer : public glass::View {
   /**
    * Handles errors when they pop up.
    */
-  void HandleGeneralError(const std::exception& e);
+  void HandleError(std::string_view msg);
 
-  /**
-   * Handles json errors when they pop up.
-   */
-  void HandleJSONError(const wpi::json::exception& e);
+  // State of the Display GUI
+  GUIState m_state;
+
+  // Stores the error display state the GUI must transition into after the error
+  // popup.
+  GUIState m_errorState;
```

Comment:
Why is this variable needed in addition to m_state?

Reply:
Yeah after further thought this wasn't needed